docs/scaffold/oracle.md

```diff
@@ -0,0 +1,238 @@
+---
+order: 7
+description: IBC oracle integration with Bandchain
+---
+
+# Bandchain Oracle Scaffold
+
+BandChain’s Oracle module is a communication module built-in compliance with IBC protocol which can query data points of various types from BandChain. In addition, other chains can ask our Oracle module for real-time information according to their needs.
+Bandchain has multiples scripts deployed into the network, and we can request any data using the script id.
+
+## IBC Module Packet Scaffold
+
+Bandchain oracles can be scaffolded only in IBC modules.
+
+To scaffold an oracle:
+
+```
+starport scaffold band [oracleName] --module [module_name]
```

Comment:
`oracleName` would imply the command scaffolds an oracle

Wouldn't be rather `queryName` or `oracleQueryName`?

Reply:
```suggestion
starport scaffold band [oracleName] --module [moduleName]
```

Reply:
I like `queryName`.